### PR TITLE
Updated assembler.py to handle org errors appropriately.

### PR DESCRIPTION
**Pull request by noelvalentin**

### CODE REVIEW

**Review by cfboy — APPROVED** [review]

File: assembler.py [hunk 1]
```diff
@@ -82,7 +82,22 @@ def store_instructions_in_ram(self):
             else:
                 if source[0].lower() == 'org':
                     # Indicates at what memory location it will begin storing instructions
+                    if len(source) > 2:
+                        # there is more than one value after the 'org' - invalid address.
+                        print('org initialization failed: too many arguments after org.')
```
Comment (ChristianPerez34) [inline assembler.py]: Change to 
```python
raise SyntaxError("Too many arguments after 'org'")
```

File: assembler.py [hunk 2]
```diff
@@ -82,7 +82,22 @@ def store_instructions_in_ram(self):
...
+                        break
```
Comment (ChristianPerez34) [inline assembler.py]: Remove this

File: assembler.py [hunk 3]
```diff
@@ -82,7 +82,22 @@ def store_instructions_in_ram(self):
...
+
                     org_address = source[1]
+                    try:
+                        value = int(org_address, 16)
+                    except ValueError:
+                        print('given hexadecimal value is invalid.')
+                        break
```
Comment (ChristianPerez34) [inline assembler.py]: Add ValueError to main.py try-except block

Use this instead `org_address = int(source[1], 16)`

File: assembler.py [hunk 4]
```diff
@@ -82,7 +82,22 @@ def store_instructions_in_ram(self):
...
+                    if int(org_address, 16) > 4096:
+                        # the number given is not within the possible values (0 to 4096).
+                        print('the number given is not within the possible values (0 to 4096).')
+                        break
```
Comment (ChristianPerez34) [inline assembler.py]: Change these lines to

```python
raise MemoryError('Exceeded memory size')
```

File: assembler.py [hunk 5]
```diff
@@ -82,7 +82,22 @@ def store_instructions_in_ram(self):
...
+
                     self.p_counter = int(org_address, 16)
                     print(f'ORG: {org_address} MEM PC: {self.p_counter}')
```
Comment (ChristianPerez34) [inline assembler.py:96]: No need for this anymore

**Review by ChristianPerez34 — CHANGES_REQUESTED** [review]

**Review by alejoreyes96 — APPROVED** [review]

**Review by ChristianPerez34 — APPROVED** [review]

**Pull request merged**